docs/installation.rst

```diff
@@ -521,6 +513,10 @@ have the same configuration.
 
     CELERY_CONFIG = CeleryConfig
 
+To start a Celery worker to leverage the configuration run: ::
+
+    celery worker --app=superset.sql_lab:celery_app -Ofair
```

Comment:
For celery the gevent flag is `-P gevent`

Reply:
Thanks. I added `--pool=gevent`. I've used the long names as they're more descriptive.